ownCloud/ownCloud/UIKit Extensions/UIColor+Extension.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UIColor+Extension.swift
+//  ownCloud
+//
+//  Created by Pablo Carrascal on 15/03/2018.
+//  Copyright © 2018 ownCloud GmbH. All rights reserved.
+//
+
+import UIKit
+
+extension UIColor {
+
+    convenience init(red: Int, green: Int, blue: Int) {
```

Comment:
For this we can try [Zeplin](https://zeplin.io/), it also works as extension of [Figma](https://www.figma.com/file/Ui4YzD3oQcxJSbzBEr80O9eU/Working-Draft-UI), we can export the frames from Figma to Zeplin and from there we can have a pixel details, sizes, colors, constraints needed and an automatic generated of UIColor extension file.
![screen shot 2018-04-02 at 12 34 42](https://user-images.githubusercontent.com/6570505/38193574-fdec4ef2-3672-11e8-94db-996fae14a58d.png)
![screen shot 2018-04-02 at 12 34 04](https://user-images.githubusercontent.com/6570505/38193581-00b43adc-3673-11e8-9c2f-62c7c0d6a44c.png)

I invited you if you want to try it, I do not know yet if it can be made open to all public too.

Reply:
Oh, nice. I knew Zeplin.io, but didn't understand what problems it solves. Does Figma data automatically refreshes in Zeplin?

Figma introduced basic "Developer Handoff" a while ago: https://help.figma.com/editor/developer-handoff

Reply:
Done

Reply:
@michaelstingl it does not automatically refreshes, but if you export again they are replaced with the previous one